arrows/qt/widgets/LoggerView.cxx

```diff
@@ -32,22 +33,20 @@ LoggerView::LoggerView(
   // Set up UI
   d->m_UI.setupUi( this );
 
-  QFont font {
-    "Monospace", 10
-  };
+  QFont font { "Monospace", 10 };
```

Comment:
Eek, this is a bug. There should be no space in `font {`. For some reason, uncrustify did not identify this as an initializer list.

Are you sure this was parsed as C++?

Reply:
Okay, it seems that while newer uncrustify isn't erroneously *adding* a space here, it also isn't *removing* the space that shouldn't be present, either. That's probably a bug, or at least a missing feature.